chess/.env.example

```diff
@@ -19,6 +19,10 @@ CHAIN_CURRENCY_DECIMALS=""
 # Example: "2.0" 
 BLOCK_TIME=""
 
+## SECURITY
+# Unique Game Security Prefix
+# Example: "MyChess"
```

Comment:
Generally I try to keep the files in the templates as detached from the template name as possible so it's easier to maintain in the future (eg. adjust a file in "chess" template and move it to the rest of the templates without accidentally adding some "chess references" to them). 

With that in mind I'd prefer `# Example: "MyGame"` so it's still the same file in all templates.

Reply:
Fixed!